src/woo_publications/publications/models.py

```diff
@@ -471,6 +471,33 @@ def gpp_app_url(self) -> str:
         return url_template.replace("<UUID>", str(self.uuid))
 
 
+class PublicationIdentifier(models.Model):
+    id: int  # implicitly provided by django
+    publicatie = models.ForeignKey(
+        Publication,
+        verbose_name=_("publication"),
+        help_text=_("The publication that this identifier identifies."),
+        on_delete=models.CASCADE,
+    )
+    kenmerk = models.CharField(
+        _("identifier"),
+        max_length=40,
+        help_text=_("The unique IRI that identifies the publication."),
+    )
+    bron = models.CharField(
+        _("source"),
+        max_length=40,
+        help_text=_("Determines where the identifier is defined and sourced from."),
+    )
+
+    class Meta:
+        verbose_name = _("kenmerk")
+        verbose_name_plural = _("kenmerken")
+
+    def __str__(self):
+        return f"{self.kenmerk} - ({self.publicatie})"
```

Comment:
This crashes if you initialize an instance without `publicatie`. In general, you should avoid using foreign key fields in the __str__` method, as there's also often a hidden performance impact (additional query).

In this case, I would simply use `f"{self.bron}: {self.kenmerk}"` as that is supposed to uniquely identify a publication anyway.